docs/60_PACKAGE.md

```diff
@@ -51,6 +51,35 @@ directory.  Defaults to the docroot.
 packages directory. This can be overridden by calling grunt package with the
 `--name` parameter.
 
+**archive**: Used to enable the `compress` option via the configuration instead
+of the command line. Defaults to false.
+
+**rsync**: If set to true, rsync will be used instead of file copying for
+improved performance. This automatically enables the `compress` option and will
+write the archive to the package destination.
+
+**vmData**: This path is used as the intermediate generation directory for the
+package when using the `rsync` option. By selecting a volume mounted in your
+VM `/data` area, this can significantly improve performance. The archive is
+still moved to the normal package destination when complete.  Defaults to
+`build/vm_data`.
```

Comment:
To make this less specific to Docker and the `/data` path set up by devtools, could this be recast as a temporary directory path (i.e. tempDir rather than vmData)? We can note in the documentation that the purpose for this path is to allow using a path that performs quickly if the project directory lives in a slow performing location (i.e. NFS).